course_info/templates/course_info/partials/details.html

```diff
@@ -6,208 +6,256 @@ <h3 class="breadcrumbs">
-                    </div>
-                    <div class="col-md-10" ng-repeat="dept in dc.courseInstance.departments">
-                         <div class="list-group" ng-if="dc.courseInstance.departments.length > 0">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="dept in dc.courseInstance.departments">
-                                {{dept.name}}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Course Groups: </strong>
-                    </div>
-                    <div class="col-md-10">
-                        <div class="list-group">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="group in dc.courseInstance.course_groups">
-                                {{group.name}}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Associated Official and Unofficial Site(s): </strong>
-                    </div>
-                    <div class="col-md-10">
-                        <div class="list-group">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="site in dc.courseInstance.sites">
-                                <div ng-switch on="site.map_type">
-                                    <div class="label label-primary col-md-1" ng-switch-when="official">Official</div>
-                                    <div class="label label-primary col-md-1" ng-switch-when="unofficial">Unofficial</div>
-                                </div>
-                                <div class="col-md-11"><a href="{{site.course_site_url}}"
-                                                          target="_blank">{{site.course_site_url}}</a>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+            <hu-editable-input
+                is-loading="dc.isDefined(dc.courseInstance.registrar_code_display)"
+                form-value="dc.courseInstance.registrar_code_display"
+                model-value="dc.formDisplayData.registrar_code_display"
+                field="registrar_code_display"
+                label="Course Code"></hu-editable-input>
 
+            <hu-field-label-wrapper
+              field="xlist_status"
+              is-loading="dc.isDefined(dc.courseInstance.xlist_status)"
+              label="Cross listing">
+                <div ng-switch on="dc.courseInstance.xlist_status">
+                  <span ng-switch-when="Primary" class="label label-info">
```

Comment:
Maybe move the `class="label label-info"` up to the switch div so you don't have to repeat it?